walkthrough.ipynb

```diff
@@ -22,24 +30,160 @@
     "\n",
-    "You'll be working in the Jupyter notebook and the platform itself is accessible via URL in the slide deck. To work with Jupyter, click \"run cell\" to run the code and see results below the cell you're currently running. "
+    "<img src=\"files.png\" alt=\"drawing\" width=\"400\"/>\n"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": null,
+   "id": "26c697c2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Lets' try running some code!\n",
+    "\n",
+    "print(\"Welcome to Lumigator!🐊\")\n",
+    "\n",
+    "# You can see the output below!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "81137051",
+   "metadata": {},
+   "source": [
+    "For more on notebooks and how cell works, check out this demo. [You can click links in cells.](https://github.com/nbgallery/Jupyter4Analysts/blob/main/J4A%20Notebook%201%20-%20Jupyter%20Syntax%20and%20Other%20Things.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1ecaa846",
+   "metadata": {},
+   "source": [
+    "## Glossary of terms \n",
+    "\n",
+    "Some terms you'll hear us using throughout the session: \n",
+    "\n",
+    "+ **Machine learning** - The process of creating a model that learns from data\n",
+    "+ **Dataset** - Data used to train models\n",
+    "+ **LLM** - Large language model, [a text-based model that performs next-word predictions](https://www.nvidia.com/en-us/glossary/large-language-models/) \n",
+    "+ **Tokens** - Words broken up into pieces to be used in an LLM \n",
+    "+ **Inference** - The process of getting a prediction from a large language model \n",
+    "+ **Embeddings** - Numerical representations of text generated by modeling \n",
+    "+ **Encoder-decoder models** - a neural network architecture comprised of two neural networks, an encoder that takes the input vectors from our data and creates an embedding of a fixed length, and a decoder, also a neural network, which takes the embeddings encoded as input and generates a static set of outputs such as translated text or a text summary\n",
+    "+ **Decoder-only models** - Receive a prompt of text directly and predict the next word\n",
+    "+ **Task** - Machine learning tasks to fit a specific model type, including translation, summarization, completion, etc. \n",
+    "+ **Ground truth** - A dataset that has been evaluated to be true by humans (or LLMs, in some cases) to be correct, that we can use as a point of comparison for our model. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0bfaf888",
+   "metadata": {},
+   "source": [
+    "# Machine Learning Workflows\n",
+    "\n",
+    "In machine learning, we are looking to generate a model artifact from data. We have several stages we care about: the data preprocessing, model training, model generation, inference, and evaluation. \n",
+    "\n",
+    "<img src=\"ml_workflow.png\" alt=\"drawing\" width=\"400\"/>\n",
```

Comment:
This workflow is mainly for model training, are you planning to update it so it shows inference / evaluation too? I am mainly trying to understand how to connect it to the following paragraph which says we focus on inference/eval